pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1589,11 +1631,76 @@ func TestPluginsConfigurationCompatibility(t *testing.T) {
 		{
 			name:             "default plugins",
 			wantPlugins:      defaultPlugins,
-			wantPluginConfig: nil,
+			wantPluginConfig: defaultPluginConfigs,
 		},
 		{
-			name:        "default plugins with customized plugin config",
-			wantPlugins: defaultPlugins,
+			name: "in-tree plugins with customized plugin config",
+			plugins: config.Plugins{
+				Filter: &config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: "NodeLabel"},
+						{Name: "ServiceAffinity"},
+					},
+				},
+				Score: &config.PluginSet{
+					Enabled: []config.Plugin{
+						{Name: "RequestedToCapacityRatio"},
```

Comment:
If we test the case `weight = 0`, then the test will fail. But as I said below, that maybe a case need to be care.